src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentByAdmissionOnlineMapper.java

```diff
@@ -200,4 +210,9 @@ private PaymentPlanSelection getPaymentPlan(CaseData caseData) {
         }
         return PaymentPlanSelection.PAY_IMMEDIATELY;
     }
+
```

Comment:
This is also duplication of code of public method uk.gov.hmcts.reform.civil.service.JudgementService#ccjJudgmentInterest

Reply:
solved